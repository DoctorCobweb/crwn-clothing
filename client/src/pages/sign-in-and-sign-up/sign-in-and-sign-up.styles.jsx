import styled from 'styled-components';

export const SignInAndSignUpContainer = styled.div`
  width: 850px;
  display: flex;
  justify-content: space-between;
  margin: 30px auto;

  @media screen and (max-width: 800px) {
    /* my attempt */
    /* display: grid;
    grid-template-columns: 1fr ;
    width: 90vw; */

    flex-direction: column;
    width: unset;
    align-items: center;

    > *:first-child {
      margin-bottom: 50px;
    }
  }
`;