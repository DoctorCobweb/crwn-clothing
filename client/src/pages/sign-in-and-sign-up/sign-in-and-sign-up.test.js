import React from 'react';
import { shallow } from 'enzyme';

import { SignInAndSignUpPage } from './sign-in-and-sign-up.component';

describe('SignInAndSignUpPage component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SignInAndSignUpPage />);
  })

  it('should render component', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  })
});