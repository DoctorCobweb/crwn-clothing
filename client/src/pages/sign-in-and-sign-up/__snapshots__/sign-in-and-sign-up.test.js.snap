// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInAndSignUpPage component should render component 1`] = `
"<SignInAndSignUpContainer>
  <Connect(SignIn) />
  <Connect(SignUp) />
</SignInAndSignUpContainer>"
`;
